etc/config-template.toml

```diff
@@ -154,6 +154,11 @@
 ## Max time to handle Coprocessor requests before timeout.
 # end-point-request-max-handle-duration = "60s"
 
+## Whether to check memory locks before handling coprocessor requests.
+## This option must be enabled if async commit is enable.
+## DO NOT use it in production. This option may be removed in the future.
+# end-point-check-memory-locks = false
```

Comment:
Why add it?

Reply:
Because coprocessor config locates in the `ServerConfig`. It is difficult to pass a storage option to coprocessor.
An alternative is to rewrite this config but it seems a bit more hacky.